### PR TITLE
docs: add release notes for Keptn 0.9.1

### PR DESCRIPTION
**Pull request by thisthat**
Signed-off-by: Giovanni Liva <giovanni.liva@dynatrace.com>

### CODE REVIEW

File: releasenotes/releasenotes_V0.9.1.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Release Notes 0.9.1
+
+
+---
```
Comment (johannes-b) [inline releasenotes/releasenotes_V0.9.1.md:4]: Please remove this line

File: releasenotes/releasenotes_V0.9.1.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+This is a bug fix release for Keptn 0.9.0, containing fixes for the Keptn services.
```
Comment (johannes-b) [inline releasenotes/releasenotes_V0.9.1.md:6]: Please change:
This is a bug fix release for Keptn 0.9.0, containing fixes for the Keptn services.

to:
This is a bug fix release for Keptn 0.9.0, containing fixes for the Bridge and Keptn core services; especially, lighthouse-service and shipyard-controller.

File: releasenotes/releasenotes_V0.9.1.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+</details>
+
+
+</p>
+</details>
+
+<details><summary>Keptn Core</summary>
+<p>
+
+- *Fixes:*
```
Comment (johannes-b) [inline releasenotes/releasenotes_V0.9.1.md:28]: Please order this group this way:  (you can copy-paste it)

```
* *lighthouse-service*: Print logs if configure monitoring fails [#5088](https://github.com/keptn/keptn/issues/5088)
* *lighthouse-service*: No error message when SLO parsing failed [#5130](https://github.com/keptn/keptn/issues/5130)
* *lighthouse-service*: Error events from the lighthouse-service are shown even though "status": "succeeded" [#5170](https://github.com/keptn/keptn/issues/5170)
* *shipyard-controller*: Returns a 500 error if a triggered event is not found [#5132](https://github.com/keptn/keptn/issues/5132)
* *shipyard-controller*: Every 10 seconds an error is produced with "could not load queued sequences" [#5138](https://github.com/keptn/keptn/issues/5138)
* *configuration-service*: deleting a file on a specific stage or for specific service not possible [#5136](https://github.com/keptn/keptn/issues/5136)
* Providing an invalid git token disables the access to the repository [#5064](https://github.com/keptn/keptn/issues/5064)
* Unable to (re)register to Keptn [#4791](https://github.com/keptn/keptn/issues/4791)
```

File: releasenotes/releasenotes_V0.9.1.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  * shipyard-controller: Returns a 500 error if a triggered event is not found [#5132](https://github.com/keptn/keptn/issues/5132)
+
+
+</p>
+</details>
+
+<details><summary>Bridge</summary>
+<p>
+
+- *Fixes:*
```
Comment (johannes-b) [inline releasenotes/releasenotes_V0.9.1.md:45]: Please remove `Bridge:` when added to the issue name.

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Pull request merged**